packages/documentation/copy/id/project-config/Configuring Watch.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Configuring Watch
+layout: docs
+permalink: /id/docs/handbook/configuring-watch.html
+oneline: Cara mengkonfigurasi mode watch TypeScript
+translatable: true
+---
+
+Compiler mendukung konfigurasi cara mengawasi file dan direktori menggunakan compiler flags di TypeScript 3.8+, dan variabel environment.
+
+## Latar Belakang
+
+Implementasi `--watch` dari compiter bergantung pada penggunaan `fs.watch` dan `fs.watchFile` yang disediakan oleh node, kedua metode ini memiliki kelebihan dan kekurangan.
+
+`fs.watch` menggunakan system event file untuk memberi tahu perubahan dalam file/direktori. Tetapi ini bergantung pada OS dan notifikasi tidak sepenuhnya dapat diandalkan dan tidak berfungsi seperti yang diharapkan pada banyak OS. Juga mungkin ada batasan jumlah watch yang dapat dibuat, misalnya. linux dan kami dapat melakukannya dengan cukup cepat dengan program yang menyertakan banyak file. Tetapi karena ini menggunakan system event file, tidak banyak siklus CPU yang terlibat. Compiler biasanya menggunakan `fs.watch` untuk melihat direktori (misalnya. Direktori sumber disertakan oleh file konfigurasi, direktori di mana resolusi modul gagal, dll.) Ini dapat menangani ketepatan yang hilang dalam memberi tahu tentang perubahan. Tetapi memantau secara rekursif hanya didukung pada Windows dan OSX. Artinya kita membutuhkan sesuatu untuk menggantikan sifat rekursif di OS lain.
```

Comment:
`menggunakan system event file` -> `menggunakan berkas _system event_`.

---

```
linux dan kami dapat melakukannya dengan cukup cepat dengan program yang menyertakan banyak file
```
Ini konteks bahasa Inggrisnya apakah?

Reply:
Teks aslinya seperti ini:

`fs.watch` uses file system events to notify the changes in the file/directory. But this is OS dependent and the notification is not completely reliable and does not work as expected on many OS. Also there could be limit on number of watches that can be created, eg. linux and we could exhaust it pretty quickly with programs that include large number of files. But because this uses file system events, there is not much CPU cycle involved. Compiler typically uses `fs.watch` to watch directories (eg. source directories included by config file, directories in which module resolution failed etc) These can handle the missing precision in notifying about the changes. But recursive watching is supported on only Windows and OSX. That means we need something to replace the recursive nature on other OS.